docs/docs/integrations/platforms/huggingface.mdx

```diff
@@ -145,3 +145,25 @@ See a [usage example](/docs/integrations/tools/huggingface_tools).
 ```python
 from langchain.agents import load_huggingface_tool
 ```
+
+## More
+
+### Hugging Face prompt injection identification
+
+>It uses one of the specialized Hugging Face models to detect prompt injection attacks.
+> It is useful for when you need to ensure that prompt is free of injection attacks.
+> Default is the `laiyer/deberta-v3-base-prompt-injection` model.  
+
+We need to install several python packages.
+
+```bash
+pip install transformers "optimum[onnxruntime]"
```

Comment:
what's the second dependency?

Reply:
I've copied it from [here](https://python.langchain.com/docs/guides/safety/hugging_face_prompt_injection).